README.md

```diff
@@ -608,7 +608,12 @@ wiggletools minI test/fixedStep.bw
 Computes the Pearson correlation between two iterators across all their points:
 
 ```
-wiggletools pearson test/fixedStep.bw test/fixedStep.bw 
+wiggletools pearson test/fixedStep.bw test/fixedStep.bw
+```
+
+Use the `strict` keyword to discard positions which have no value, instead of using the default value (see section on Default Values below):
```

Comment:
This use of strict was not detailed for other n-ary operators, so let's not create an anti-pattern.

Reply:
Alright! I was thinking it could be useful to have a list of which operators accept "strict" arguments, and which are strict by default. Do you think that's necessary?